config/prow/config_knative.yaml

```diff
@@ -151,6 +151,18 @@ presubmits:
         - "./test/presubmit-integration-tests-latest-release.sh"
       dot-dev: true
 
+  knative/client-contrib:
```

Comment:
To be consistent, it's better to also add the simple periodics part for this client-contrib repo, see the setting for client - https://github.com/knative/test-infra/blob/810ad47815885a5f2ceee52539d4aef4ec4486d5/config/prow/config_knative.yaml#L444